### PR TITLE
Move exection of sql scripts from sql container into ANET application

### PR DESCRIPTION
**Pull request by VassilIordanov**
implements nci-agency/anet#662

### CODE REVIEW

File: insertBaseData.sql [hunk 1]
```diff
@@ -712,7 +712,7 @@ INSERT INTO approvalSteps (uuid, name, advisorOrganizationUuid)
 INSERT INTO approvers (approvalStepUuid, positionUuid)
 	VALUES ((SELECT uuid from approvalSteps where name = 'Default Approvers'), (SELECT uuid from positions where name = 'ANET Administrator'));
 
--- Set approvalStepUuid's from organizations with default
+-- Set approvalStepUuids from organizations with default
```
Comment (gjvoosten) [inline insertBaseData.sql]: Please remove this change from the PR, it is unrelated (and *'s* with apostrophe is generally used as the plural form throughout the comments).

File: src/main/java/mil/dds/anet/DatabaseScriptCommand.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		try { 
+			final String sqlScript = new String(Files.readAllBytes(Paths.get(sqlFilePath))); 
+			jdbiHandle = factory.build(environment, configuration.getDataSourceFactory(), "database-script").open();
+			jdbiHandle.createScript(sqlScript).execute();
+		} 
+		catch (IOException e) { 
+			e.printStackTrace(); 
+		}
+		finally {
+			if (jdbiHandle != null)
```
Comment (gjvoosten) [inline src/main/java/mil/dds/anet/DatabaseScriptCommand.java]: Use braces around `if` body.

File: insertBaseData.sql (hunk 1 above)
Comment (VassilIordanov) [reply insertBaseData.sql]: The reason I made this change is because the ` in a comment confuses the jdbi script parser. With it the script will not load.

Comment (gjvoosten) [reply insertBaseData.sql]: Hah, that would seem to me to be a bug in the parser (any character should be allowed in a comment, quotes needn't be balanced; it looks suspiciously like https://github.com/jdbi/jdbi/issues/75 ), but I'll give in .

**Pull request merged**